.github/workflows/test.yml

```diff
@@ -62,10 +62,18 @@ jobs:
           platform: linux-x64-gnu
       - name: Build
         shell: bash
-        env:
-          CARGO_BUILD_TARGET: ${{ steps.neon.outputs.target }}
-          NEON_BUILD_PLATFORM: linux-x64-gnu
```

Comment:
Removing this because the default works fine and allows the integration tests to use the `index.node` file built from `npm run debug`. If we set this, Neon won't use the name `index.node` for the artifact and `@neon-rs/load` will look for the platform-specific file (and fail in CI) instead of loading the debug `index.node`. Relevant code in `neon dist`: https://github.com/neon-bindings/neon-rs/blob/b92c01b4c297ed34ab90348c641b181f1a5b0b8b/src/cli/src/commands/dist.ts#L65-L83.